stylesheets/README.md

```diff
@@ -0,0 +1,3 @@
+# New File
+
+When adding a new stylesheet to the repository, currently, after merge, it must be added to https://liquipedia.net/commons/Special:ResourceLoaderArticles
```

Comment:
Might wanna add something about bumping ResourceLoaderCacheVersion after merge for changes to take effect?

Reply:
Done